### DISCUSSION TITLE
The watchers_count returned by the Github Rest api repo interface is the same as stargazers_count

### DISCUSSION THREAD

**Discussion by fatjyc** [community/community#70980] [Apps, API and Webhooks] [answered]
### Select Topic Area

Bug

### Body

I use GIthub Rest api to get repository information.

But the return value watches_count is the same as stargazers_count.

When I look at the web page, there are different. So I think there are some problems here.

<img width="852" alt="image" src="https://github.com/community/community/assets/4507629/312ac810-29b2-4b98-bd92-2148cdf10e3b">
https://api.github.com/repos/babelcloud/LLM-RGB
<img width="906" alt="image" src="https://github.com/community/community/assets/4507629/6ea90701-55b2-498c-a0cb-359f38060aed">
https://github.com/babelcloud/LLM-RGB

**Comment by mootari** [comment, marked as answer]
Turns out that's by design: https://github.com/orgs/community/discussions/24795#discussioncomment-3245470
